README.md

```diff
@@ -10,97 +10,151 @@
+- Save changes
+- Click 'Refresh' in the bottom right corner
 
-If you start a new Jina instance, click "try again" and it will connect.
+<p align="center">
+<img src=".github/.README_images/refresh.png?raw=true" alt="log server settings" width="40%">
+</p>
 
-Connection status will be indicated by the globe icon. A red X means the dashboard is not connected to any Jina instance, a green checkmark means there is an active connection.
+  - The globe icon in the top right corner will indicate connection status. A red X means the Dashboard is not connected to any Jina instance; a green checkmark indicates an active connection. The icon should change to green to show Dashboard, and the Jina Deamon are now connected.
 
 <p align="center">
 <img src=".github/.README_images/2859cc17.png?raw=true" alt="log server settings" width="80%">
 </p>
 
-You should now see the log stream and flow visualization.
-
-## Self-host Dashboard
+ - You should now be able to see the log stream and flow visualization.
 
-One can self-host a dashboard locally.
+# Self-host Dashboard
+You can also self-host a dashboard instance locally.
 
-1. `git clone https://github.com/jina-ai/dashboard.git && cd dashboard`.
-2. Install dependencies using command `npm i`.
-3. Run dashboard via the following ways .
+Note: features like the Hub and GitHub login will not work when running the Dashboard locally. They are restricted to the `dashboard.jina.ai` origin. They are not necessary to view logs or interact with flows. If you would like to browse the Hub, do so from [dashboard.jina.ai](https://dashboard.jina.ai/).
 
-Note: features like the hub and GitHub login will not work when running locally as they are restricted to the `dashboard.jina.ai` origin. They are not necessary to view logs or interact with flows. If you would like to browse the hub do so from [dashboard.jina.ai](https://dashboard.jina.ai/).
+## Set up
+- Clone the GitHub repo
+```bash
+git clone https://github.com/jina-ai/dashboard.git && cd dashboard
+```
+- Install dependencies using command
+```bash
+npm i
+```
+## Run dashboard
 
 ### Run in debug mode
 
-1. `npm run start_dev-server`
+- Start the testServer
+```bash
+npm run start_dev-server
+```
+- testServer should now be running on `http://localhost:5000` by default
 
-    testServer will be running on `http://localhost:5000` by default
-2. `npm run start-dashboard`
+- Start the dashboard
+```bash
+npm run start-dashboard
+```
 
-    Dashboard will be served on `http://localhost:3000` by default
+- Dashboard should now be served on `http://localhost:3000` by default
 
 ### Run in the live mode
 
-1. `npm run build-dashboard`
-2. `node dashboard`
-3. dashboard will be served on `http://localhost:3030` by default
+- Build dashboard:
+```bash
+npm run build-dashboard
+```
+- Run dashboard:
+```bash
+node dashboard
+```
+- Dashboard should now be served on `http://localhost:3030` by default
 
-## Contributing
+# Contributing
```

Comment:
Can you add the https://github.com/jina-ai/jina/#community  + https://github.com/jina-ai/jina/#open-governance and https://github.com/jina-ai/jina/#join-us section from the main repo please? Just copy and paste!